src/app/plan/duck/sagas.ts

```diff
@@ -66,6 +66,26 @@ function* getPlanSaga(planName) {
     throw err;
   }
 }
+function* patchPlanSaga(planValues) {
+  const state = yield select();
+  const migMeta = state.migMeta;
+  const client: IClusterClient = ClientFactory.hostCluster(state);
+  try {
+    const getPlanRes = yield call(getPlanSaga, planValues.planName);
+    const updatedMigPlan = updateMigPlanFromValues(getPlanRes.data, planValues);
```

Comment:
Danil did something similar with his PR -- I think the only thing that should be in the payload should be the patched value, not the rest of the object that the conversion returns. It was written for PUT, so it returns everything. We only want to include a JSON merge patch body that updates the closed value to true.

https://tools.ietf.org/html/rfc7386

Reply:
@eriknelson just updated